heartbeat/IPaddr2

```diff
@@ -471,6 +473,14 @@ Consider the resource failed if the interface has status DOWN or LOWERLAYERDOWN.
 <shortdesc lang="en">Consider the resource failed if the interface has status DOWN or LOWERLAYERDOWN</shortdesc>
 <content type="string" default="${OCF_RESKEY_check_link_status_default}"/>
 </parameter>
+
+<parameter name="kill_connections_on_stop">
+<longdesc lang="en">
+Kill connections using 'ss -K src IP' before deleteing interface
```

Comment:
Typo in "deleting", and should say "before and after" based on the logic in the stop function.